### PR TITLE
Feature/utility

### PR DESCRIPTION
**Pull request by elsafluss**
* Is this a fix or a feature?
feature
* What is the change/fix?
webpack is working
the data api is being called successfully
* Where should the reviewer start?
util.js
* How should this be tested?
start servers, open localhost:8080
* Screenshots(if appropriate)

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        </button>
+        <label for='search-input' class='hidden'>Type search query here</label>
+        <input id='search-input' class='search-input' type='text' placeholder='Search recipes by name or ingredients' value=''>
+      </form> -->
+    </header>
+    <main>
+      <section class='all-cards'>
+      </section>
+    </main>
+    <script type="text/javascript" src="./main.bundle.js"></script>
```
Comment (DrewBradley) [inline dist/index.html:29]: I am so glad we got this figured out. We are going to conquer this assignment!

Comment (colefiscus) [inline dist/index.html:29]: Glad we figured this out! Eventually.

File: src/index.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <meta charset='UTF-8'>
     <title>Whats Cookin</title>
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap' rel='stylesheet'>
+    <link rel="stylesheet" href="styles.css">
```
Comment (DrewBradley) [inline src/index.html:7]: So much prettier now!

File: src/util.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+export const getData = (api) => {
```
Comment (DrewBradley) [inline src/util.js:1]: Having a utility file will make it SO DRY!

**Review by DrewBradley — APPROVED** [review]

File: src/util.js (hunk 3 above)
Comment (colefiscus) [inline src/util.js:1]: Great job figuring out the export language.

**Review by colefiscus — APPROVED** [review]
We got things to work!

**Pull request merged**